modules/Bio/EnsEMBL/IO/Format.pm

```diff
@@ -322,7 +364,7 @@ sub validate_as_integer {
 
 sub validate_as_floating_point {
   my ($self, $value) = @_;
-  return $value =~ /^-?\d+\.?\d*$/ ? undef : "Value $value is not a floating point number";
+  return $value =~ /^-?\d+\.?\d*$/ ? 1 : 0;
```

Comment:
Same problems as for the integer patters plus the possibility of different character being used as decimal separator. Same conclusion.